backend/src/services/review.ts

```diff
@@ -141,6 +149,20 @@ export async function findReviewsForAccessRequests(accessRequestIds: string[]) {
   return reviews.filter((review) => requiredRoles.accessRequest.includes(review.role))
 }
 
+export async function findReviewForAccessRequest(accessRequestId: string) {
+  const review = await Review.findOne({ accessRequestId: accessRequestId })
```

Comment:
This isn't a big issue right now, but preferably I think it would be good if this process of deleting reviews could handle having multiple reviews associated with a single access request.